gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -134,6 +135,21 @@ class ComposePlugin : Plugin<Project> {
                 }
             }
         }
+
+        checkAndWarnAboutUsingJsPlugin(project)
+    }
+
+    private fun checkAndWarnAboutUsingJsPlugin(project: Project) {
+        val msg = "Project has kotlin(\"js\") plugin applied.\n" +
+                "It's necessary to apply kotlin(\"multiplatform\") plugin " +
+                "in order to use \"org.jetbrains.compose\" plugin with k/js targets.\n" +
+                "See https://github.com/JetBrains/compose-jb/blob/master/tutorials/Web/README.md"
+
+        project.plugins.all {
```

Comment:
Usually `plugins.withId` is used for that case: 
```
plugins.withId("org.jetbrains.kotlin.js") {
    // ...
}
```

Reply:
thanks. fixed